packages/wallet-backend/src/rafiki/integration/webhook/webhook.service.ts

```diff
@@ -0,0 +1,247 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
```

Comment:
I think fixing and correctly parsing the received event should be done. Is this eslint rule for that?

Reply:
This eslint exclude is a leftover, thx